<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Stat</class>
 <widget class="QDialog" name="Stat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Загруженность аэропорта</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tw_statTab">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="yearTab">
      <attribute name="title">
       <string>Загруженность аэропорта за год</string>
      </attribute>
      <widget class="QWidget" name="wid_yearWorkload" native="true">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>461</width>
         <height>371</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="monthTab">
      <attribute name="title">
       <string>Загруженность аэропорта за месяц</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QComboBox" name="cb_monthList"/>
       </item>
       <item>
        <widget class="QWidget" name="wid_monthWorkload" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pb_close">
     <property name="text">
      <string>Закрыть</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pb_close</sender>
   <signal>clicked()</signal>
   <receiver>Stat</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>186</x>
     <y>280</y>
    </hint>
    <hint type="destinationlabel">
     <x>219</x>
     <y>1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
